### PR TITLE
Add LICENSE file to the docker image

### PR DESCRIPTION
**Pull request by nan-yu**

### CODE REVIEW

**Comment by nan-yu** [comment]
Need to recreate the PR against release-3.x branch.

**Comment by thockin** [comment]
If you really want to be nice you'd do both release-3.x and master :)

File: Dockerfile.in [hunk 1]
```diff
@@ -87,4 +87,8 @@ WORKDIR /tmp
 
 # Just dump the binary in the root.
 ADD bin/{ARG_OS}_{ARG_ARCH}/{ARG_BIN} /{ARG_BIN}
+
+# Add License file.
+COPY LICENSE LICENSE
```
Comment (thockin) [inline Dockerfile.in:92]: can you be explicit and say `/LICENSE` or even `/LICENSE.git-sync.txt`

Comment (nan-yu) [reply Dockerfile.in:92]: I actually need all third-party licenses. I added a make target to generate the license files and added them to the image. see PR: #358

**Comment by nan-yu** [comment]
> If you really want to be nice you'd do both release-3.x and master :)

Sure. I'll make another PR one #358 is merged.

**Pull request closed without merging**